api/envoy/service/discovery/v3/discovery.proto

```diff
@@ -18,9 +18,15 @@ option (udpa.annotations.file_status).package_version_status = ACTIVE;
 
 // [#protodoc-title: Common discovery API components]
 
+message ResourceLocator {
```

Comment:
Comments on the message field, and perhaps some contrived examples to help the reader?